build-system/global-configs/canary-config.json

```diff
@@ -22,6 +22,7 @@
   "story-ad-placements": 1,
   "story-load-first-page-only": 0.1,
   "story-disable-animations-first-page": 0.1,
+  "amp-story-desktop-one-panel": 1,
```

Comment:
Nit: Changes to `canary-config` and `prod-config` shouldn't be necessary anymore.